api/src/main/java/com/eleks/academy/pharmagator/dataproviders/DummyProviderImpl.java

```diff
@@ -21,6 +21,7 @@ private MedicineDto buildDto(int i) {
                 .externalId(String.valueOf(i))
                 .title("title" + i)
                 .price(BigDecimal.valueOf(Math.random()))
+                .pharmacy("Dummy")
```

Comment:
Please, extract pharmacies names into configuration.
At the same time, Dummy one may be eliminated.